src/bindings/python/src/openvino/_ov_api.py

```diff
@@ -62,6 +62,8 @@ def __getattr__(self, name: str) -> Any:
         return getattr(self.__model, name)
 
     def clone(self) -> "Model":
+        if self.__model is None:  # added checks to see before clonning
```

Comment:
The comment doesn't seem meaningful.
```suggestion
        if self.__model is None:
```